docs/content/en/getting_started/upgrading.md

```diff
@@ -84,7 +84,9 @@ notifications but also for sending the reset password emails. It is highly recom
 value if you are satisfied. If you installed DefectDojo earlier, you can expect `"from@example.com"` there. A fresh
 installation will use `"no-reply@example.com"`
 
+This release [updates](https://github.com/DefectDojo/django-DefectDojo/pull/5450) our helm dependencies. There is a breaking change if you are using the mysql database from the helm chart because we replaced the deprecated chart from the stable repo with a chart from bitnami. If you have persistance enabled, ensure to backup your data before upgrading. All data get lost when replacing the mysql chart during the upgrade. For data migration take a look at the mysql backup and restore process.
```

Comment:
Have you tested upgrade? What will happen if you define existing Claim  `primary.persistence.existingClaim`?

Reply:
Upgrade with default values works. I don't know what happens with an existing claim.

Reply:
Sorry I was not testing it, but when you upgrade existing (generated from stable) mysql to one from bitnami, will data be preserved, or it will create completely new (empty) PVC? 

So my question is, in case if it will not use same PVC (new one will be created) and prior upgrade you specify exitingClaim name (one that is created from stable) what will happen?  Just wondering, in case it works it can be use for data migration, even I'm fully with you about the statement of backup/restore. 

I'm reading you statement that all data will be lost, if persistence is not enable data will be lost anyhow.

Reply:
I don't know. I wrote that the data will be lost to warn people and to assume the worst case - maybe there is a way to prevent it. Anyway I don't think people are running a productive defectdojo with the mysql chart (it is deprecated for a while now!). Most will use an external database. And if there is someone, it will be easier to restore your data from the backup than testing such a migration scenario. But feel free to write a migration guide :)

Reply:
That is ok.